Shared/Samples/Show labels on layer/ShowLabelsOnLayerView.swift

```diff
@@ -0,0 +1,111 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ShowLabelsOnLayerView: View {
+    /// A map with a light gray canvas basemap centered on the USA.
+    private var map = {
+        let map = Map(basemapStyle: .arcGISLightGrayBase)
+        map.initialViewpoint = Viewpoint(
+            center: Point(x: -10626699.4, y: 2150308.5),
+            scale: 74016655.9
+        )
+        return map
+    }()
+    
+    /// A Boolean value indicating whether to show an alert.
+    @State private var isShowingAlert = false
+    
+    /// The error shown in the alert.
+    @State private var error: Error? {
+        didSet { isShowingAlert = error != nil }
+    }
+    
+    var body: some View {
+        // Create a map view to display the map.
+        MapView(map: map)
+            .task {
+                do {
+                    // Create a feature table from the URL.
+                    let featureTable = ServiceFeatureTable(url: .usaCongressionalDistricts)
+                    
+                    try await featureTable.load()
```

Comment:
No need to explicitly load, but I'm OK with keeping it in fear that the data might be unavailable one day.